modules/registry-configuring-storage-azure-user-infra.adoc

```diff
@@ -27,6 +27,6 @@ $ oc edit configs.imageregistry.operator.openshift.io/cluster
 
 storage:
   azure:
-    accountName: <account-name>
+    storage_account_name: <account-name>
```

Comment:
```suggestion
    accountName: <storage-account-name>
```

Reply:
Thank you! I thought that looked odd initially. :)